src/containers/SuperAdminSuperAdmin.tsx

```diff
@@ -0,0 +1,177 @@
+    setConfirmRemoveSuperAdminDialog
+  ] = useState<boolean>(false);
+
+  const {
+    data: superadmins,
+    loading,
+    refetch: refetchSuperAdmins
+  } = useGetSuperAdminsQuery();
+  const [editSuperAdminStatus] = useEditSuperAdminStatusMutation();
+
+  const classes = useStyles();
+
+  const handleRemoveSuperadmin = (email: string) => {
+    setConfirmRemoveSuperAdminDialog(true);
+    setRemoveSuperAdminEmail(email);
+  };
+
+  const handleAddSuperAdmin = async (email: string) => {
+    await editSuperAdminStatus({
+      variables: {
+        superAdminStatus: true,
+        userEmail: email
+      }
+    });
+    refetchSuperAdmins();
+  };
+
+  const handleAddSuperAdminDialog = () => {
+    setAddSuperAdminDialog(true);
+  };
+
+  const handleConfirmRemoveSuperAdminDialogClose = () => {
+    setConfirmRemoveSuperAdminDialog(false);
+  };
+
+  const handleConfirmRemoveSuperAdminDialogSubmit = async () => {
+    await editSuperAdminStatus({
+      variables: {
+        superAdminStatus: false,
+        userEmail: removeSuperAdminEmail as string
+      }
+    });
+    setConfirmRemoveSuperAdminDialog(false);
+    refetchSuperAdmins();
+  };
+
+  const handleCloseSuperAdminDialog = () => setAddSuperAdminDialog(false);
+
+  const columns: GridColDef[] = [
+    {
+      field: "displayName",
+      headerName: "Name",
+      flex: 4
+    },
+    {
+      field: "email",
+      headerName: "Email",
+      flex: 4
+    },
+    {
+      field: "removeSuperAdmin",
+      headerName: "Remove Superadmin",
+      sortable: false,
+      flex: 2,
+      renderCell: (params: GridRenderCellParams) => (
+        <IconButton
+          onClick={(event) => {
+            event.stopPropagation();
+            handleRemoveSuperadmin(params.row.email);
+          }}
+        >
+          <DeleteForeverIcon />
+        </IconButton>
+      )
+    }
+  ];
+  return loading ? (
+    <div>Loading...</div>
+  ) : (
+    <div>
+      <Paper>
+        <DataGridPro
+          autoHeight
+          columns={columns}
+          rows={superadmins?.superadmins ?? []}
+          pagination
+          pageSize={pageSize}
+          rowsPerPageOptions={[10, 30, 50]}
+          onPageChange={(newPageSize) => setPageSize(newPageSize)}
+        />
+      </Paper>
+      <Fab
+        size="large"
+        variant="extended"
+        color="primary"
+        className={classes.fab}
+        onClick={handleAddSuperAdminDialog}
+      >
+        <AddIcon /> Add Superadmin
+      </Fab>
+      <AddSuperAdminDialog
+        open={addSuperAdminDialog}
+        onClose={handleCloseSuperAdminDialog}
+        onSubmit={handleAddSuperAdmin}
+      />
+      <Dialog
+        open={confirmRemoveSuperAdminDialog}
+        onClose={handleConfirmRemoveSuperAdminDialogClose}
+      >
+        <DialogTitle>Confirm remove superadmin</DialogTitle>
+        <DialogContent>
+          <DialogContentText>
+            Are you sure you want to remove the superadmin?
```

Comment:
Suggestion to mimic the Add Superadmin Dialog if possible
```suggestion
            Are you sure you want to remove the superadmin ({user.displayName} [{user.email}])?
```